types/lodash/tslint.json

```diff
@@ -6,6 +6,7 @@
         "max-line-length": [false],
         "no-unnecessary-generics": false,
         "typedef-whitespace": [false],
-        "unified-signatures": false
+        "unified-signatures": false,
+        "use-default-type-parameter": false
```

Comment:
I faced the error below when I ran `npm run lint lodash` without this line.

```
$ npm run lint lodash

> definitely-typed@0.0.3 lint /Users/tsuchiya/.ghq/github.com/tuttieee/DefinitelyTyped
> dtslint types "lodash"

The 'use-default-type-parameter' rule threw an error in '/Users/tsuchiya/.ghq/github.com/tuttieee/DefinitelyTyped/types/lodash/lodash-tests.ts':
TypeError: Cannot read property 'default' of undefined
    at checkArgsAndParameters (/Users/tsuchiya/.ghq/github.com/tuttieee/DefinitelyTyped/node_modules/tslint/lib/rules/useDefaultTypeParameterRule.js:63:19)
    at cb (/Users/tsuchiya/.ghq/github.com/tuttieee/DefinitelyTyped/node_modules/tslint/lib/rules/useDefaultTypeParameterRule.js:52:13)
    at visitNode (/Users/tsuchiya/.ghq/github.com/tuttieee/DefinitelyTyped/node_modules/typescript/lib/typescript.js:17487:24)
    at Object.forEachChild (/Users/tsuchiya/.ghq/github.com/tuttieee/DefinitelyTyped/node_modules/typescript/lib/typescript.js:17738:24)
    at cb (/Users/tsuchiya/.ghq/github.com/tuttieee/DefinitelyTyped/node_modules/tslint/lib/rules/useDefaultTypeParameterRule.js:54:19)
    at visitNodes (/Users/tsuchiya/.ghq/github.com/tuttieee/DefinitelyTyped/node_modules/typescript/lib/typescript.js:17496:30)
    at Object.forEachChild (/Users/tsuchiya/.ghq/github.com/tuttieee/DefinitelyTyped/node_modules/typescript/lib/typescript.js:17727:24)
    at cb (/Users/tsuchiya/.ghq/github.com/tuttieee/DefinitelyTyped/node_modules/tslint/lib/rules/useDefaultTypeParameterRule.js:54:19)
    at visitNodes (/Users/tsuchiya/.ghq/github.com/tuttieee/DefinitelyTyped/node_modules/typescript/lib/typescript.js:17496:30)
    at Object.forEachChild (/Users/tsuchiya/.ghq/github.com/tuttieee/DefinitelyTyped/node_modules/typescript/lib/typescript.js:17727:24)
```

Reply:
The ideal solution would be to fix `loadsh-tests.ts` instead, but if that isn't trivial then this is fine as a temporary solution.

Reply:
Right, this is workaround.
I took a glance at `lodash-tests.ts` but it seemed tough to fix it though I just wanted to start development.

Reply:
The same error has been reported: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/40548#issuecomment-558069037

This error seems a problem of the tslint rule (use-default-type-parameter)'s implementation, and tslint is now deprecated and won't be fixed. Then, I think disabling this rule is a good compromise unless DefinitelyTyped decides to introduce eslint instead of tslint.